lib/queryBuilder/operations/UpsertGraphAndFetchOperation.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const DelegateOperation = require('./DelegateOperation');
+const RelationExpression = require('../RelationExpression');
+const UpsertGraphOperation = require('./UpsertGraphOperation');
+
+class UpsertGraphAndFetchOperation extends DelegateOperation {
+
+  constructor(name, opt) {
+    super(name, opt);
+
+    if (!this.delegate.is(UpsertGraphOperation)) {
+      throw new Error('Invalid delegate');
+    }
+  }
+
+  get models() {
+    return this.delegate.models;
+  }
+
+  get isArray() {
+    return this.delegate.isArray;
+  }
+
+  onAfter3(builder) {
```

Comment:
@koskimas I needed to use `onAfter3()` here instead of `onAfter2()`, because `UpsertGraphOperation` already defines `onAfter2()`.